sqlx-core/src/query_builder.rs

```diff
@@ -635,4 +653,23 @@ mod test {
             "SELECT * FROM users WHERE id = 99"
         );
     }
+
+    #[test]
+    fn test_query_builder_with_args() {
+        let mut qb: QueryBuilder<'_, Postgres> = QueryBuilder::new("");
+
+        let query = qb
+            .push("SELECT * FROM users WHERE id = ")
+            .push_bind(42i32)
+            .build();
+
+        let mut qb: QueryBuilder<'_, Postgres> =
+            QueryBuilder::new_with(query.sql(), query.take_arguments());
```

Comment:
This test isn't actually running because `sqlx-core` doesn't have a `postgres` feature anymore.

I'll have to migrate this to be an integration test or something.

Reply:
Okay, is this a blocker?